tests/test_ctl.py

```diff
@@ -646,3 +647,22 @@ def test_reinit_wait(self, mock_get_dcs):
             result = self.runner.invoke(ctl, ['reinit', 'alpha', 'other', '--wait'], input='y\ny')
         self.assertIn("Waiting for reinitialize to complete on: other", result.output)
         self.assertIn("Reinitialize is completed on: other", result.output)
+
+
+class TestPatronictlPrettyTable(unittest.TestCase):
+
+    def setUp(self):
+        self.pt = PatronictlPrettyTable(' header', ['foo', 'bar'], hrules=ALL)
+
+    def test__get_hline(self):
+        self.pt._hrule = '+-----+-----+'
+        self.assertEqual(self.pt._hrule, '+ header----+')
```

Comment:
Don't we want to check the second 'getter' call?

Reply:
Here we go: https://github.com/zalando/patroni/pull/2436/commits/acfbc70c15d3ab3e8270fbca69383349a06a82d4